woocommerce/wc-admin
# Pause inbox message "GivingFeedbackNotes"

Fixes #6697

This PR pauses the inbox message "GivingFeedbackNotes". To get that, we remove the note from the `possibly_add_notes` list in `src/Events.php`.

### Screenshots

![screenshot-one wordpress test-2021 04 13-17_12_17](https://user-images.githubusercontent.com/1314156/114614942-8159f980-9c7b-11eb-97dd-94cd3d2a7a0b.png)


### Detailed test instructions:

_As these changes won't be tested by our GlobalStep partners, the testing instructions described here are more technical than usual._

-   Check out this branch and run in your terminal:
```
npm run test:zip
```
- Create a brand new ephemeral site (you can use JN for that).
- Change your store activation date to 10 days old. You can use [this plugin](https://wordpress.org/plugins/wp-phpmyadmin-extension/) to have `phpMyAdmin` on the site to run a SQL query like this:
```
UPDATE `wp_options` SET `option_value`=UNIX_TIMESTAMP(DATE_SUB(NOW(), INTERVAL 10 day)) WHERE `option_name` = 'woocommerce_admin_install_timestamp';
```

-   Run the cron (you can use the plugin [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/) for that)
-   Verify the note `wc-admin-store-notice-giving-feedback-2` hasn't been added. If you want to check directly in the DB you should use a SQL query like:
```
SELECT * FROM `wp_wc_admin_notes` WHERE `name` = 'wc-admin-store-notice-giving-feedback-2';
```
